project/build.scala

```diff
@@ -206,7 +207,10 @@ object build extends Build {
         case ex: Exception if ex.getMessage.contains(nothingToCommit) => println(nothingToCommit)
       }
     },
+    // TODO: doesn't work at the moment, see https://github.com/sbt/sbt-pgp/issues/42
+    publishSigned := publish.value,
```

Comment:
I usually have a separate project that aggregates all my publishable stuff. Often I have integration tests or other random things that I don't want to publish either